services/connected_account_dev.yaml

```diff
@@ -0,0 +1,221 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file is copy of `connected_account.yaml` with `dev` values in its configuration map